src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
+Over the course of the past decade, search engines, particularly Google, have continued to push towards becoming the presentation layer of the web. These advancements are partially driven by their improved ability to extract information from unstructured content (e.g., [passage indexing](https://blog.google/products/search/search-on/)) and the adoption of semantic markup in the form of _structured data_. Search engines have encouraged content creators and developers to implement structured data to give more visibility to their content within components of search results.
 
-In a move from “strings to things” , search engines have agreed upon a broad vocabulary of objects in support of marking up a variety of people, places, and things within web content. However, only a subset of that vocabulary triggers inclusion within search results components. Google specifies those that they support and how they’re displayed in their [“Search Gallery”](https://developers.google.com/search/docs/guides/search-gallery), and provides [a tool](https://search.google.com/test/rich-results) to validate their support and implementation. 
+In a move from ["strings to things"](https://blog.google/products/search/introducing-knowledge-graph-things-not/), search engines have agreed upon a broad vocabulary of objects in support of marking up a variety of people, places, and things within web content. However, only a subset of that vocabulary triggers inclusion within search results components. Google specifies those that they support and how they're displayed in their [search gallery](https://developers.google.com/search/docs/guides/search-gallery), and provides [a tool](https://search.google.com/test/rich-results) to validate their support and implementation. 
 
-As search engines evolve to reflect more of these elements in search results, the incidence rates of the different  vocabulary changes across the web.
+As search engines evolve to reflect more of these elements in search results, the incidence rates of the different vocabularies change across the web.
 
-As part of our examination, we took a look at the incidence rates of different types of structured markup. The available formats include [RDFa](https://www.w3.org/TR/rdfa-primer/) and [Schema.org](https://schema.org/) which come in both the microformats and [JSON-LD](https://www.w3.org/TR/json-ld11/) flavors. Google has recently [dropped the support for data-vocabulary](https://developers.google.com/search/blog/2020/01/data-vocabulary), a vocabulary that was primarily used to implement breadcrumbs. 
+{# TODO(authors): Is schema.org itself a "format"? #}
```

Comment:
@aleyda

Reply:
@rviscomi I really can't think of another way to call it - could you or maybe @Tiggerito suggest one?

Reply:
schema.org is a vocabulary, it defines what can exist in its world. Formats are things like json-ld, microdata and rdfa which are ways to write information in a vocabulary. data-vocabulary is an alternate vocabulary that Google is dropping. 

Chiseling in a stone is a format, the hieroglyphics you make are the vocabulary.

This said after a few 🍷s. So don't quote me, unless it works.

There are more formats. Those are the ones I looked for because they are the main ones that Google officially cares about. 

In the future I think it would be worth looking into other things like open graph (a vocabulary+rdfa format) and its hybrid/proprietary use by Facebook, Twitter and Pinterest.

Reply:
@Tiggerito omg yes! they're vocabularies :) Will update accordingly @rviscomi! 

Tony, you always bring "structured data light", with wine or without :D